src/main/java/com/ericsson/eiffel/remrem/semantics/clone/ChangeProperties.java

```diff
@@ -0,0 +1,203 @@
+package com.ericsson.eiffel.remrem.semantics.clone;
```

Comment:
Can this class be rewritten? It feels very unlike java principles with all public static method.